src/main/java/io/xpire/logic/parser/SetReminderCommandParser.java

```diff
@@ -31,7 +34,12 @@ public SetReminderCommand parse(String args) throws ParseException {
                     SetReminderCommand.MESSAGE_USAGE), e);
         }
 
-        String threshold = args.split("\\|")[1];
+        try {
+            threshold = ParserUtil.parseReminderThreshold(args.split("\\|")[1]);
+        } catch (IllegalValueException e) {
+            throw new ParseException(String.format(MESSAGE_INVALID_REMINDER_THRESHOLD,
```

Comment:
What are the cases where this will be thrown?

Reply:
check|1|hello or check|2|-5

Reply:
Icic, what about set reminder|1 and set reminder|1|?

Reply:
Gd point. Not cached. Will fix that.